webhooks/apps/v1/petset.go

```diff
@@ -0,0 +1,151 @@
+/*
+Copyright AppsCode Inc. and Contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1
+
+import (
+	"context"
+	"fmt"
+	api "kubeops.dev/petset/apis/apps/v1"
+	"kubeops.dev/petset/pkg/features"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	appsv1 "k8s.io/api/apps/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/utils/ptr"
+	ctrl "sigs.k8s.io/controller-runtime"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/webhook"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+)
+
+// SetupPetSetWebhookWithManager registers the webhook for PetSet in the manager.
+func SetupPetSetWebhookWithManager(mgr ctrl.Manager) error {
+	return ctrl.NewWebhookManagedBy(mgr).For(&api.PetSet{}).
+		WithValidator(&PetSetCustomWebhook{mgr.GetClient()}).
```

Comment:
Looks like we only need the mutator, not validator here.

Reply:
https://github.com/kubeops/installer/blob/master/charts/petset/templates/webhook-server/validating_webhook.yaml

Yes, no usage.  But I checked, Validating webhook is getting created here. 
So, remove from both place ? 

May be we can keep them as-it-is. exposing a valition path is not a big deal.